lib/rubocop/cop/style/empty_case_condition.rb

```diff
@@ -77,10 +77,14 @@ def correct_multiple_alternative_whens(corrector, when_nodes)
 
             # In `when a; r` we have two children: [a, r].
             # In `when a, b, c; r` we have 4.
-            next unless children.count > 2
+            next unless children.size > 2
 
-            corrector.insert_before(children.first.loc.expression, '[')
-            corrector.insert_after(children[-2].loc.expression, '].any?')
+            range =
+              Parser::Source::Range.new(when_node.loc.expression.source_buffer,
+                                        children[0].loc.expression.begin_pos,
+                                        children[-2].loc.expression.end_pos)
+
+            corrector.replace(range, children[0..-2].map(&:source).join(' || '))
```

Comment:
Good idea. For some reason I didn't even think about dropping the last `child` on creation. I will give it a shot.